.circleci/config.yml

```diff
@@ -15,16 +15,16 @@ jobs:
       - checkout
 
       # Download and cache dependencies
-      - restore_cache:
-          keys:
-          - v3-dependencies-{{ checksum "package.json" }}
+      # - restore_cache:
```

Comment:
@treygriffith probably didnt mean to check this in

Reply:
I did in fact mean to check this in because circle cache was failing. I'm considering removing the cache altogether. Talk me off the edge.

Reply:
@treygriffith lol we have had all kinds of issue with the caching. Looks like the best we can do right now is:

1. make sure npm is updated (6.9.x)
2. remove the lock and push the change
3. add the lock file back and push the change